bpf/bpf_lxc.c

```diff
@@ -756,6 +770,11 @@ int tail_handle_ipv4(struct __ctx_buff *ctx)
 		return send_drop_notify(ctx, SECLABEL, dstID, 0, ret,
 					CTX_ACT_DROP, METRIC_EGRESS);
 
+#ifdef ENABLE_CUSTOM_CALLS
+	encode_custom_prog_meta(ctx, ret, dstID, CUSTOM_CALLS_IDX_EGRESS);
+	tail_call_static(ctx, &CUSTOM_CALLS_MAP, CUSTOM_CALLS_IDX_EGRESS);
```

Comment:
Any thoughts on IPv6 support?

Reply:
I plan to add it, and I believe it will be mostly similar to IPv4, with the hooks to be added to the same points in the datapath. For that reason I have only focused on IPv4 so far, to make it easier to review and address feedback. I intend to add it as a new commit next time I push to this PR.